wdio.browserstack.config.js

```diff
@@ -0,0 +1,62 @@
+const config = {
+  hostname: 'hub.browserstack.com',
+  port: 443,
+  baseUrl: '',
+  user: process.env.BROWSERSTACK_USER,
+  key: process.env.BROWSERSTACK_KEY,
+
+  specs: ['./e2e/test.js'],
+  maxInstances: 10,
+
+  // Capabilities Generator
+  // https://www.browserstack.com/automate/capabilities?tag=selenium-4
+  commonCapabilities: {
+    'browserstack.debug': true,
+    'browserstack.video': true,
+    'browserstack.console': 'verbose',
+    'browserstack.networkLogs': true,
+  },
+
+  capabilities: [
```

Comment:
What in the time impact of adding multiple other combinations?

Reply:
It depends of the offer on the test service.

With an offer like [Automate Pro on BrowserStack](https://www.browserstack.com/accounts/subscriptions) we can run 5 tests in parallel, so 5 combinations in total without impacting the test duration.